src/main/java/com/dawn/jat/illuminati/post/controller/PostController.java

```diff
@@ -38,4 +37,23 @@
         }
         return new ResponseEntity<>(postEntity, HttpStatus.OK);
     }
-}
+
+    /**
+     * Gets list of Post Summary.
+     * @throws PostSummaryNotFoundException if Post Summary has No Content
+     */
+    @GetMapping(value = "/all-post/summary")
+    public ResponseEntity<Object> getAllPostSummary() {
+        List<PostSummaryEntity> allPost = postService.findPostSummary();
+
+        Map<String, Object> body = new LinkedHashMap<>();
+
+        if (allPost.isEmpty()) {
+            throw new PostSummaryNotFoundException("Cannot find post summary");
+        }
+
+        body.put("data", allPost);
```

Comment:
Map<String, Object> body = new LinkedHashMap<>();
 is too far from here

Reply:
Updated